lib/auth/tun.go

```diff
@@ -734,22 +741,25 @@ func (c *TunClient) getClient() (*ssh.Client, error) {
 	return nil, trace.Wrap(err)
 }
 
-func (c *TunClient) dialAuthServer(authServer utils.NetAddr) (*ssh.Client, error) {
+func (c *TunClient) dialAuthServer(authServer utils.NetAddr) (sshClient *ssh.Client, err error) {
 	config := &ssh.ClientConfig{
 		User: c.user,
 		Auth: c.authMethods,
 	}
-	client, err := ssh.Dial(authServer.AddrNetwork, authServer.Addr, config)
-
-	if err != nil {
-		log.Infof("TunDialer: ssh.Dial: %v", err)
-
+	const dialRetryInterval = time.Duration(time.Millisecond * 50)
```

Comment:
also, I think that `50` ms is too aggressive. Over 5 attempts is't just quarter of a second worth of retries, usually network issues don't get resolved in such a short period